src/Buffer.mo

```diff
@@ -48,7 +52,9 @@ module {
     if (oldCapacity == 0) {
       1;
     } else {
-      Prim.abs(Prim.floatToInt(Prim.floatCeil(Prim.intToFloat(oldCapacity) * INCREASE_FACTOR)));
+      // calculates ceil(oldCapacity * INCREASE_FACTOR) without floats
+      let product = oldCapacity * INCREASE_FACTOR_NUME;
+      (product / INCREASE_FACTOR_DENOM) + (if (product % INCREASE_FACTOR_DENOM != 0) { 1 } else { 0 })
```

Comment:
```suggestion
      (product + INCREASE_FACTOR_DENOM - 1 ) / INCREASE_FACTOR_DENOM
```
Might do the trick as well. @ggreif amirite? I very well may not be

Reply:
@crusso yep this should work. Example: `((57 * 7) + 3 - 1) / 3` gives `133.7`, `57 * 7/3 = 133`.

Reply:
Okay will swap out